packages/management/addons-manager/bundle/config/values.yaml

```diff
@@ -0,0 +1,12 @@
+#@data/values
+#@overlay/match-child-defaults missing_ok=True
+
+---
+tanzuAddonsManager:
+  namespace: tanzu-system
+  createNamespace: true
+  deployment:
+    hostNetwork: false
```

Comment:
question: could you put some comments on why this is set to false ? I think we set hostNetwork:true for addons manager ?

Reply:
@vijaykatam can you confirm this should be false or true?

Reply:
should be true

Reply:
updated to true